src/qt/rpcconsole.h

```diff
@@ -58,14 +58,14 @@ class RPCConsole: public QWidget
         CMD_ERROR
     };
 
-    enum TabTypes {
-        TAB_INFO = 0,
-        TAB_CONSOLE = 1,
-        TAB_GRAPH = 2,
-        TAB_PEERS = 3
+    enum class TabTypes {
+        INFO = 0,
+        CONSOLE = 1,
+        GRAPH = 2,
+        PEERS = 3
     };
```

Comment:
Why not just

```
enum class TabTypes { INFO, CONSOLE, GRAPH, PEERS };
```

?

Reply:
Could be, but note that almost all are multi-line - which is better for adding comments - so I'm going to leave as it is.